docs/source/markdown/podman-network-create.1.md

```diff
@@ -29,6 +31,11 @@ Driver to manage the network (default "bridge").  Currently on `bridge` is suppo
 Define a gateway for the subnet. If you want to provide a gateway address, you must also provide a
 *subnet* option.
 
+**--host-device**
+
+Host networking device that will be used by *macvlan* connections.  This option must be used with the
```

Comment:
Should this be plural, since you can add one or more devices for a vlan?

Reply:
you can only add one.  shall i singularize the language in the man page. is that what confused this?

Reply:
The code allows users to add multiple. Should we combine the options, into one.  
--macvlanaddr="1.2.3.4"
Which would indicate that you want to use macvlan and don't have to specify the address separately. removes extra error checking.